pkg/operation/care/garden_health.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Should be rename `pkg/operation/care/health.go` -> `pkg/operation/care/shoot_health.go` to match the pattern?